app/controllers/api/v2/access_requests_controller.rb

```diff
@@ -10,11 +10,18 @@ def approve
         render status: 200, json: { result: result }
       end
 
+      def show
+        authorize AccessRequest
+        @access_request = AccessRequest.find(params[:id])
+
+        render jsonapi: @access_request, include: [:requester]
+      end
+
       def index
         authorize AccessRequest
-        @access_requests = AccessRequest.requested
+        @access_requests = AccessRequest.requested.includes(:requester).order(request_date_utc: :asc)
```

Comment:
🎉 the ordering will hopefully fix the flaky test

Reply:
Bonus points for a scope i.e. `by_request_date` rather than ordering in the controller